packages/my-extended-retail-app/pwa-kit-overrides/app/hooks/use-einstein.js

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import _useEinstein, {EinsteinAPI as _EinsteinAPI} from '^retail-react-app/app/hooks/use-einstein'
+
+export class EinsteinAPI extends _EinsteinAPI {
+    async sendSomeEvent() {
+        console.log('--- Einstein: sending some event')
+    }
+}
+
+const useEinstein = () => {
+    const einstein = _useEinstein(EinsteinAPI)
+
+    return {
+        ...einstein,
+        async sendViewPage(...args) {
+            console.log(
+                '--- can send pageview event to other analytics providers, by piggy backing on existing implementation'
```

Comment:
You can choose to send an event to another analytics providers by piggy backing on this Einstein hook. But I'd suggest to have a separate file for it. Because those other providers probably would behave differently and have different event names than Einstein. 

For example: while Einstein is more API-based and more plumbing required, Google Analytics (GA) is more plug-and-play. GA typically asks you insert a script tag into your html head. Then it would automatically send events like page view for you.